src/scripts/ci.zig

```diff
@@ -31,10 +33,22 @@ pub fn main(shell: *Shell, gpa: std.mem.Allocator, cli_args: CliArgs) !void {
     if (cli_args.validate_release) {
```

Comment:
Out of scope for this PR, but right now we have a `--language` filter -- I wonder if it would make more sense to have a `--task=<tests|readmes>` (or replace `--language` with a more general `--filter`) so that you can generate the READMEs without running all of the client tests.

Reply:
Yeah, the current filtering is a bit odd indeed. But my general rule of thumb for these sorts of dev-tools is to do pain-driven development --- send a specific PR for a specific feature once you need it for a particular workflow. These things _shouldn't_ be costly to change, so they could use less up-front design. 

I often find that just commenting specific stuff out manually works better than trying to recall elaborate filtering syntax >_<